src/main/java/gumtree/spoon/builder/LabelFinder.java

```diff
@@ -140,7 +140,7 @@ public void visitCtComment(CtComment comment) {
 
 	@Override
 	public <T extends Annotation> void visitCtAnnotation(CtAnnotation<T> annotation) {
-		label = annotation.toString();
+		label = annotation.getType().getQualifiedName();
```

Comment:
Hold the review.

This might break other things not covered in test cases. Example,
```diff
import java.lang.annotation.ElementType;
import java.lang.annotation.Target;

@Target({ElementType.TYPE})
@interface Column {
    int id();
-    String a();
+    String b();
    String type() default "String";
    String value() default "42"
}
```

Reply:
@monperrus you can review this PR now.

I have added the test case for the above, but it was already broken. See #154 . In short, we currently calculate the diff only between **the first type** in the file.